fe/fe-core/src/test/java/org/apache/doris/analysis/ShowDbStmtTest.java

```diff
@@ -17,16 +17,16 @@
 
 package org.apache.doris.analysis;
 
-import org.junit.Assert;
```

Comment:
Please update your IDE code style setting, don't introduce this unnecessary change.

Reply:
@kangkaisen

Reply:
I mean all xxxTest files, not only this test file.

Reply:
@kangkaisen  all done.  please review it again